Islandora/islandora_mirador
# update-mirador-pat --- wild attempt at documentation, mainly focusing…

… on text overlay

# What does this Pull Request do?

Greatly expands islandora_mirador documentation

* **Related GitHub Issue**: https://github.com/Islandora/islandora_mirador/issues/4

* **Other Relevant Links**: https://github.com/Islandora/islandora_mirador/pull/14

# What's new?
* Adds a "Text Overlay" section to the README.md doc which covers in excruciating detail with numerous screenshots the things that need to be set up in order to get Mirador text overlay to work.
* Some clarification on the Mirador library choices.
* Rewrites some other sections a bit.
* Random word-smithing

# How should this be tested?

* Review the how-to steps to make sure I haven't made a mistake or utilized incorrect assumptions.
* Make suggestions for improving the clarity of what I've written.

# Documentation Status

* This is a documentation PR.

# Additional Notes:
Any additional information that you think would be helpful when reviewing this
 PR.

# Interested parties
@alxp
